src/test/xeroClient.spec.ts

```diff
@@ -100,6 +100,7 @@ describe('the XeroClient', () => {
       expect(xeroClient).toHaveProperty('payrollAUApi')
       expect(xeroClient).toHaveProperty('payrollUKApi')
       expect(xeroClient).toHaveProperty('payrollNZApi')
+      expect(xeroClient).toHaveProperty('appStoreApi')
```

Comment:
Are there any other low hanging fruit / sanity check tests we could add around the app store API test coverage?

Reply:
This is the lowest of fruit haha. There is really only 1 new function being added. Setting up http://getsandbox.com/ for node sdk would be a great addition though